src/Models/Exceptions/MappingException.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Middleware.Models.Exceptions;
+
+[Serializable]
+public class MappingException : Exception
+{
+    public Type TypeFrom { get; set; }
```

Comment:
Remove `set` accessors from `TypeFrom` and `TypeTo` properties

Reply:
Removed